docs/app-deployment-sample.md

```diff
@@ -0,0 +1,164 @@
+# Deploying An Application
+
+The following set of steps show how to deploy an application to a kubernetes environment using the Open Liberty's Helm based operator.
+
+## Before you start:
+Install the following:
+* Git
+* Docker.
+* Kubectl.
+* Kubernetes environment (OpenShift, Minikube, etc).
+
+The sample application used here is modresorts-1.0.war, which is part of the app modernization GIT repository (https://github.com/IBM/appmod-resorts). Clone this repository:
+
+`git clone https://github.com/IBM/appmod-resorts.git`
+
+### 1. Package your application in a docker container and publish it.
+
+a. Go the app modernization repository previously cloned
+
+`cd appmod-resorts`
+
+b. Create a docker file to install the application.
+
+Dockerfile:
+
+```
+FROM open-liberty:javaee8
```

Comment:
I am wondering if we should pick a different (smaller) image.  I'm guessing this app doesn't need EE 8 features, just a web container?

Reply:
I tried it with webProfile8 and it still worked but the startup time was not significantly faster.  I think you should leave it alone.  Just ignore me.  =P